nuxt-pinia-tailwind/schemes/authScheme.ts

```diff
@@ -40,21 +40,28 @@ export default class CustomLoginScheme extends LocalScheme {
       // Fetch stars
       const starsEndpoint: HTTPRequest = {
         ...endpoint,
-        url: `users/${user.login}/starred`,
+        url: `users/${user.login}/starred?per_page=1`,
+        headers: {
+          Accept: 'application/vnd.github.preview',
+        },
       };
 
       const getStars = this.$auth.requestWith(this.name, starsEndpoint);
       //
 
-      const [{ data: orgs }, { data: stars }] = await Promise.all([
-        getOrgs,
-        getStars,
-      ]);
+      const [orgs, stars] = await Promise.all([getOrgs, getStars]);
+
+      const starsCount =
+        stars.headers?.link?.split(',')[1].match(/.*page=(?<page_num>\d+)/)
```

Comment:
What is we made this a re-useable function. This function can be where other helper functions are if there be any.

```ts
const pageCounter = (value: string) => value?.split(',')[1].match(/.*page=(?<page_num>\d+)/)?.groups?.page_num || 0;

const starsCount = pageCounter(stars.headers?.link);
const orgsCount = pageCounter(orgs.headers?.link);
```

Reply:
This is nice! It helps to make the call quicker instead of just calling the whole thing. I agree with @hdJerry, we should make this a utility function though